<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
hover{
color: red;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="field_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_run">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ЗАПУСТИТЬ</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="field_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="save_output">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Сохранить отчёт"/>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border: 2px  gray;}
QPushButton::hover{ background-color: #8399ff;}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_input">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border: 2px  gray;}
QPushButton::hover{ background-color: #8399ff;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_input">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border: 2px  gray;}
QPushButton::hover{ background-color: #8399ff;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_output">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border: 2px  gray;}
QPushButton::hover{ background-color: #8399ff;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="input_from_file">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border: 2px  gray;}
QPushButton::hover{ background-color: #8399ff;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="info">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>-10</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border: 2px  gray;}
QPushButton::hover{ background-color: #8399ff;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_all">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border: 2px  gray;}
QPushButton::hover{ background-color: #8399ff;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border: 2px  gray;}
QPushButton::hover{ background-color: #8399ff;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Поле ввода:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Поле вывода:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>copy_input</sender>
   <signal>clicked()</signal>
   <receiver>field_input</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copy_input</sender>
   <signal>clicked()</signal>
   <receiver>field_input</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copy_output</sender>
   <signal>clicked()</signal>
   <receiver>field_output</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copy_output</sender>
   <signal>clicked()</signal>
   <receiver>field_output</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
